testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/AdminClientUtil.java

```diff
@@ -145,6 +164,42 @@ public static ResteasyClient createResteasyClient(boolean ignoreUnknownPropertie
         return resteasyClientBuilder.build();
     }
 
+    public static ResteasyClient createResteasyClientWithKeystoreAndTruststore(boolean ignoreUnknownProperties, Boolean followRedirects) throws CertificateException, NoSuchAlgorithmException, KeyStoreException, IOException, KeyManagementException {
+        ResteasyClientBuilder resteasyClientBuilder = (ResteasyClientBuilder) ResteasyClientBuilder.newBuilder();
+
+        if ("true".equals(System.getProperty("auth.server.ssl.required"))) {
+            File truststore = new File(PROJECT_BUILD_DIRECTORY, "containers/auth-server-undertow/keycloak.truststore");
+            try {
+                resteasyClientBuilder.sslContext(getSSLContextWithTruststoreAndKeystore(
+                  truststore, "secret",
+                  new File(PROJECT_BUILD_DIRECTORY, "containers/auth-server-undertow/client.jks"), "secret"));
+            } catch (UnrecoverableKeyException e) {
+                throw new RuntimeException(e);
+            }
+
+            System.setProperty("javax.net.ssl.trustStore", truststore.getAbsolutePath());
+        }
+
+        // We need to ignore unknown JSON properties e.g. in the adapter configuration representation
+        // during adapter backward compatibility testing
+        if (ignoreUnknownProperties) {
```

Comment:
Is it possible to remove this since `ignoreUnknownProperties` is always false when you are calling it? Will be also nice to remove `ignoreUnknownProperties` parameter of this method entirely.

I think that you copy/pasted this from existing test right? I think it is obsolete and can be removed everywhere, but you don't need to care about other occurences of this (I will possibly create follow-up issue for that).